samples/subsys/input/input_dump/prj.conf

```diff
@@ -0,0 +1,4 @@
+CONFIG_LOG=y
```

Comment:
sample doesn't seem to log, just printks.

Reply:
I actually just added it recently cause I hit a priority issue and was getting no fault message at all with no logging for some reason, am I missing something? There used to be a compact fault message all the time. Anyway let's get rid of this.